Client/public/locales/en/student.json

```diff
@@ -0,0 +1,10 @@
+{
+    "student": {
+        "1": "Filter to Find Supporter",
```

Comment:
I think we should use more descriptive names here than numbers
```suggestion
        "findSupporter": "Filter to Find Supporter",
```

Reply:
Fixed